node/service/src/chain_spec/pangoro.rs

```diff
@@ -94,7 +94,9 @@ pub fn genesis_config() -> ChainSpec {
 					array_bytes::hex_into_unchecked(account),
 					GenesisAccount {
 						balance: (MANY_COINS * (10 as Balance).pow(9)).into(),
-						..Default::default()
```

Comment:
why?

Reply:
Compared with [GenesisAccount Common](https://github.com/darwinia-network/darwinia-common/blob/main/frame/dvm/evm/src/lib.rs#L559), the [GenesisAccount Frontier](https://github.com/darwinia-network/frontier/blob/darwinia-v0.12.4/primitives/evm/src/lib.rs#L66-L77) doesn't have default implementation.